src/earn/EarnEnterAmount.tsx

```diff
@@ -659,14 +687,40 @@ function FeeDetailsBottomSheet({
             )}
           </View>
         </View>
-        <RowDivider style={{ marginBottom: Spacing.Large32 }} />
-        <View style={styles.gap8}>
+        <RowDivider />
```

Comment:
nit: can we also remove the style prop from RowDivider now that it's not used? Would prefer to keep overrides at a minimum where possible

Reply:
Removed in d6d56d3139fe51ddf5661ffbb870b4399b1d49a2!